icons/rollup/rollup-line-wordmark.svg

```diff
@@ -0,0 +1 @@
61a.372.372 0 0 1 .372-.369zm16.553 2.471c.204.001.408.012.611.031l.067.006c1.5.153 2.95.807 4.097 1.837a7.818 7.818 0 0 0-3.23-1.11h-.001l-.057-.005a7.674 7.674 0 0 0-.423-.033h-.006a6.472 6.472 0 0 0-.654.006 5.622 5.622 0 0 0-1.208.208c-1.003.297-1.834.907-2.256 1.896-.681 1.588-.108 3.259.819 4.486a6.051 6.051 0 0 0 1.558 1.46c.047.029.092.05.139.077l.019.012.074.04a3.165 3.165 0 0 0 .595.256c.112.033.222.056.329.067l.015.002.02.001c.052.004.102.006.152.005l.01-.001c.056-.002.112-.007.165-.017h.001c.068-.012.131-.029.192-.05h.001l.002-.001a.99.99 0 0 0 .415-.278c.144-.167.226-.379.266-.613.074-.443 0-.972-.133-1.505.093.188.247.461.299.598.188.492.249.845.194 1.159-.003.018-.005.037-.009.055a1.34 1.34 0 0 1-.12.322c-.219.424-.708.867-1.486 1.522-.778.654-1.841 1.521-3.199 2.821-2.725 2.609-5.463 6.648-7.626 10.094-1.081 1.723-2.018 3.297-2.734 4.468-.358.585-.661 1.07-.898 1.42v.001c-.118.175-.22.316-.302.418a1.004 1.004 0 0 1-.168.174c-.642.367-1.684.458-2.379.02l-.004.001c-.004-.004-.007-.007-.022-.017l-.002-.001a1.507 1.507 0 0 1-.128-.095l-.025-.02a1.495 1.495 0 0 1-.157-.15 3.447 3.447 0 0 1-.242-.294c-.135-.181-.21-.414-.274-.613a2.155 2.155 0 0 1-.049-.283v-.001c-.044-.386-.017-.905.135-1.622v-.002c.155-.726.451-1.67.921-2.865.144-.357.28-.7.456-1.105a78.986 78.986 0 0 1 2.103-4.411 124.822 124.822 0 0 1 1.461-2.769v-.001l.002-.003v-.001a783.6 783.6 0 0 1 2.284-4.17c.502-.908.988-1.786 1.424-2.558v-.001l.001-.001.003-.006c1.013-1.79 1.86-3.217 2.604-4.365.055-.085.108-.163.162-.245.179-.272.353-.526.521-.765l.17-.241c.208-.289.41-.559.604-.8l.001-.001.002-.002a11.804 11.804 0 0 1 .917-1.009c.196-.187.387-.35.576-.492a4.43 4.43 0 0 1 1.136-.627 4.434 4.434 0 0 1 .737-.197c.079-.015.157-.031.239-.043l.004-.001c.152-.021.309-.036.469-.047.26-.017.53-.027.82-.026zm1.046.99c1.705.071 3.551.809 4.706 1.99a10.61 10.61 0 0 1 1.375 5.242c0 3.973-2.163 7.433-5.373 9.277l-.001.001a.637.637 0 0 0-.247.837l5.553 10.992a.37.37 0 0 1-.331.54H5.884c.014-.016.026-.028.04-.047v-.001c.09-.112.195-.258.315-.436.234-.346.529-.819.874-1.381l.031-.048c.717-1.173 1.653-2.745 2.732-4.464 2.158-3.438 4.894-7.464 7.585-10.04 1.35-1.293 2.405-2.154 3.186-2.81.781-.656 1.294-1.106 1.55-1.604.064-.124.11-.251.139-.382l.001-.002c.087-.394.018-.829-.19-1.377-.276-.726-.795-1.675-1.538-3.077l-.004-.008-.001-.002-.003-.001-.003-.005a.135.135 0 0 0-.108-.07.134.134 0 0 0-.117.055s-.904 1.275-2.227 3.172c-.05-.061-.102-.118-.149-.181-.479-.634-.853-1.382-.998-2.169a3.529 3.529 0 0 1-.032-.206 3.478 3.478 0 0 1 .242-1.847c.536-1.254 1.783-1.86 3.231-1.945a6.87 6.87 0 0 1 .626-.003zm-.61 3.522c.125.253.525 1.08.805 2.067.163.575.261 1.157.206 1.594-.028.218-.092.398-.197.529a.718.718 0 0 1-.467.259 1.015 1.015 0 0 1-.272.006h-.002l-.003-.001a1.73 1.73 0 0 1-.293-.062l-.048-.016a2.616 2.616 0 0 1-.364-.147 3.673 3.673 0 0 1-.454-.267c-.036-.024-.072-.05-.108-.076a5.97 5.97 0 0 1-.933-.854c1.242-1.779 2.032-2.892 2.13-3.032zM2.408 76.877l.001.009-.001.008v-.017z"/></g></svg>
```

Comment:
Can we make the lines a bit thicker for the logo?

I also wonder if it would look better with hollow text, so the whole icon is lines. What do you think?
This is not a must though. In fact most line-wordmark icons don't have hollow/line text.
I will accept it even if the text is unchanged :)

If you wanna go for hollow text, you can use the djangorest PR as an example: https://github.com/devicons/devicon/pull/1407/files#diff-75a0ca900d5867593762acd10b934d39e826fe5c00a097f0e93a2f9a6380b53a

Examples of hollowed text:
Djangorest: https://github.com/devicons/devicon/pull/1407/files#diff-75a0ca900d5867593762acd10b934d39e826fe5c00a097f0e93a2f9a6380b53a
Unity: https://github.com/devicons/devicon/blob/develop/icons/unity/unity-line-wordmark.svg

Reply:
Sure, I'll create a sample. :+1:

Reply:
How's this, @Snailedlt?
![test](https://user-images.githubusercontent.com/41539361/200621078-f8cbf77d-b106-4488-a0b3-fd1b91c83112.svg)

Reply:
@Snailedlt done